tsconfig.json

```diff
@@ -20,6 +20,7 @@
     "types": ["jest"]
   },
   "files": [
-    "src/index.ts"
+    "src/index.ts",
+    "examples/index.tsx"
```

Comment:
Running `npm run build` will also build the examples with this change, we don't want to do that.

Reply:
The reason why I added this was for the examples files, the ts checking seemed not working in vs code file editor. I must miss something here.